tests/pytorch/r1.12/common.libsonnet

```diff
@@ -30,12 +30,11 @@ local volumes = import 'templates/volumes.libsonnet';
   PyTorchXlaDistPodTest:: common.PyTorchXlaDistPodTest + r1_12,
   PyTorchGkePodTest:: common.PyTorchGkePodTest + r1_12,
   Functional:: mixins.Functional {
-    schedule: '0 7 * * *',
+    schedule: '0 7 * * 1',
     tpuSettings+: {
       preemptible: false,
     },
   },
-  Convergence:: mixins.Convergence,
```

Comment:
If you set the schedule field to null, then we won't generate any of the conv tests without having to remove them from the repository yet:

```
Convergence:: mixins.Convergence {
  schedule: null
}
```

Reply:
This is very helpful! Updated.